flasgger/utils.py

```diff
@@ -251,6 +252,36 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def __replace_ref(schema: dict, relative_path: str):
+    absolute_path = os.path.dirname(sys.argv[0])
+    new_value = {}
+    for key, value in schema.items():
+        print('key:', key)
```

Comment:
@isaacfi Could you delete the print statements? Thanks

Reply:
Yes of course I forgot to delete them